persistent_chains/testnet-2/genesis.json

```diff
@@ -0,0 +1,197 @@
+{
+  "genesis_time": "2021-09-24T14:47:05.733203Z",
+  "chain_id": "cheqd-testnet-2",
```

Comment:
Can we just call this cheqd-testnet?

Reply:
Discussed in slack: decided to go with `cheqd-testnet-2`